Library/Formula/bento4.rb

```diff
@@ -0,0 +1,31 @@
+class Bento4 < Formula
+  desc "Full-featured MP4 format and MPEG DASH library and tools"
+  homepage "https://www.bento4.com"
+  url "https://codeload.github.com/axiomatic-systems/Bento4/tar.gz/v1.4.3-607"
```

Comment:
Use https://github.com/axiomatic-systems/Bento4/archive/v1.4.3-607.tar.gz and it'll autodetect the `version`